scripts/scil_compute_tracking_dipy.py

```diff
@@ -0,0 +1,265 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Local streamline HARDI tractography.
+
+The tracking direction is chosen in the aperture cone defined by the
+previous tracking direction and the angular constraint.
+
+Algo 'eudx': the peak from the spherical function (SF) most closely aligned
+to the previous direction.
+Algo 'det': the maxima of the spherical function (SF) the most closely aligned
+to the previous direction.
+Algo 'prob': a direction drawn from the empirical distribution function defined
+from the SF.
+"""
+
+import argparse
+import logging
+
+from dipy.core.sphere import HemiSphere
+from dipy.data import SPHERE_FILES, get_sphere
+from dipy.direction import (DeterministicMaximumDirectionGetter,
+                            ProbabilisticDirectionGetter)
+from dipy.direction.peaks import PeaksAndMetrics
+from dipy.tracking.local import BinaryTissueClassifier, LocalTracking
+from dipy.tracking.streamlinespeed import length, compress_streamlines
+from dipy.tracking import utils as track_utils
+
+import nibabel as nib
+from nibabel.streamlines import Field
+from nibabel.orientations import aff2axcodes
+from nibabel.streamlines.tractogram import LazyTractogram
+import numpy as np
+
+from scilpy.reconst.utils import (find_order_from_nb_coeff,
+                                  get_b_matrix, get_maximas)
+from scilpy.tracking.tools import get_theta
+from scilpy.io.utils import (create_header_from_anat,
+                             add_overwrite_arg, add_sh_basis_args, add_verbose,
+                             assert_inputs_exist, assert_outputs_exists)
+
+DETERMINISTIC = 'det'
+PROBABILISTIC = 'prob'
+EUDX = 'eudx'
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        description=__doc__,
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
```

Comment:
Change class to `RawTextHelpFormatter`, else the formatting of the help is broken.